development/ts-migration-dashboard/files-to-convert.json

```diff
@@ -40,8 +40,6 @@
   "app/scripts/controllers/permissions/selectors.test.js",
   "app/scripts/controllers/permissions/specifications.js",
   "app/scripts/controllers/permissions/specifications.test.js",
-  "app/scripts/controllers/preferences.js",
-  "app/scripts/controllers/preferences.test.js",
```

Comment:
Wow, great catch! I see that this file is very outdated (there's even network controller which is long gone from extension)